mcs/class/System.Core/common_System.Core.dll.sources

```diff
@@ -98,6 +98,10 @@ System.Security.Cryptography/SHA512CryptoServiceProvider.cs
 ../../../external/corefx/src/System.Collections/src/System/Collections/Generic/HashSetEqualityComparer.cs
 ../../../external/corefx/src/System.Collections/src/System/Collections/Generic/ICollectionDebugView.cs
 
+
+../../../external/corefx/src/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs
```

Comment:
Not sure it should be in System.Core but `DispatchProxyGenerator` uses System.Linq.